packages/brain/src/modules/cron/trackValidatorsPerformance.ts

```diff
@@ -0,0 +1,167 @@
+import { BeaconchainApi } from "../apiClients/beaconchain/index.js";
+import { PostgresClient } from "../apiClients/postgres/index.js";
+import logger from "../logger/index.js";
+import { ValidatorStatus } from "../apiClients/types.js";
+import { BlockProposalStatus } from "../apiClients/postgres/types.js";
+import { BrainDataBase } from "../db/index.js";
+
+const logPrefix = "[CRON - trackValidatorsPerformance]: ";
+
+// TODO: at this moment Lighthouse client does not support retrieving:
+// - liveness of validator from finalized epoch:
+// ```400: BAD_REQUEST: request epoch 79833 is more than one epoch from the current epoch 79835```
+// - sync committee rewards:
+// ```404: NOT_FOUND: Parent state is not available! MissingBeaconState(0xa9592014ad4aa3d5dcc4ef67b669278a85fb4dbe80f12364f2486444b7db3927)```
+
+/**
+ * Cron task that will track validators performance for the epoch finalized and store it in the Postgres DB.
+ * If any issue is arisen during the process, it will be retried after 1 minute. If the issue persists until the epoch
+ * finalized changes, the issue will be logged and stored in the DB.
+ *
+ * @param validatorPubkeys - The pubkeys of the validators to track.
+ * @param postgresClient - Postgres client to interact with the DB.
+ * @param beaconchainApi - Beaconchain API client to interact with the Beaconchain API.
+ * @param minGenesisTime - The minimum genesis time of the chain.
```

Comment:
I would rename this to genesisTimestamp, `minGenesisTime` makes it look as if there were multiple genesis times IMO